### DISCUSSION TITLE
AW4C keeps asking for update

### DISCUSSION THREAD

**Discussion by Alfa-Sakazaki** [thatskymod/Sky-CotL-Scripts#38] [General]
Please update AW4C, because when I want to use it there it says I have to update it first

**Comment by thatskymod** [comment]
# Fixed in libdukun.so
Please note that the traditional AW4C has reached its end-of-life and will no longer receive official support or updates. As a replacement, consider using libdukun, the successor to AutoWax, which includes a number of tweaks and improvements. Libdukun is located in the Canvas Libs folder within this repository.

**Comment by J1R3NG0D** [comment]
libdukun is also out of date, I tested it today.
